common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -230,15 +233,30 @@ define('WordCloudMain', [], function () {
         $.each(response.student_words, function (word, stat) {
             var percent = (response.display_student_percents) ? ' ' + (Math.round(100 * (stat / response.total_count))) + '%' : '';
 
-            studentWordsKeys.push('<strong>' + word + '</strong>' + percent);
+            studentWordsKeys.push(HtmlUtils.interpolateHtml(
+                gettext('{startTag}{word}{endTag}{percent}'),
+                {
+                    startTag: HtmlUtils.HTML('<strong>'),
+                    word: word,
+                    endTag: HtmlUtils.HTML('</strong>'),
+                    percent: percent
+                }
+            ));
         });
         studentWordsStr = '' + studentWordsKeys.join(', ');
```

Comment:
What I meant was that you could replace the "$.each(response.student_words, function (word, stat)" loop and studentWordsKeys with a usage of HtmlUtils.joinHtml. But looking at this further, it wouldn't simplify the code much because you still want to deal with putting commas in-between the individual words (which can have internationalization impact, but that's an existing bug).